### PR TITLE
Fix real data value handling

### PR DESCRIPTION
**Pull request by RRozak**
I fixed parsing of floating point values with the function used by the original validator.

### CODE REVIEW

File: docs/CONTRIBUTORS [hunk 1]
```diff
@@ -76,3 +76,4 @@ Yossi Nivin
 Yuri Victorovich
 Yutetsu TAKATSUKASA
 Yves Mathieu
+Ryszard Różak
```
Comment (kbieganski) [inline docs/CONTRIBUTORS]: Names in `CONTRIBUTORS` should be listed in alphabetical order.

File: src/UhdmAst.cpp [hunk 2]
```diff
@@ -153,8 +154,14 @@ AstNode* get_value_as_node(vpiHandle obj_h) {
     }
     case vpiRealVal: {
         std::string valStr = std::to_string(val.value.real);
-        V3Number value(value_node, valStr.c_str());
-        value_node = new AstConst(new FileLine("uhdm"), value);
+	
+        bool parseSuccess;
+        double value = VString::parseDouble(valStr, &parseSuccess);
+        assert(parseSuccess && "Unable to parse real value");
```
Comment (kbieganski) [inline src/UhdmAst.cpp]: Please use a Verilator-style assert:
https://github.com/antmicro/verilator/blob/8a8996ae5eb9afeb4dc998b9f406089ad2c87f78/src/V3EmitC.cpp#L2823

**Review by kbieganski — CHANGES_REQUESTED** [review]

File: src/UhdmAst.cpp (hunk 2 above)
Comment (RRozak) [reply src/UhdmAst.cpp]: Thank you, done

File: docs/CONTRIBUTORS (hunk 1 above)
Comment (RRozak) [reply docs/CONTRIBUTORS]: I didn't know about it. I fixed it, thank you.

**Review by kbieganski — APPROVED** [review]

**Pull request merged**